src/DynamoCore/Graph/Nodes/CustomNodes/Function.cs

```diff
@@ -74,19 +74,27 @@ public void UpdatePortsForUnresolved(PortModel[] inputs, PortModel[] outputs)
         /// <summary>
         /// The unique id of the underlying function.
         /// </summary>
-        public Guid FunctionUuid
+        public Guid FunctionSignature
         {
             get { return Definition.FunctionId; }
         }
 
+        public string FunctionType
```

Comment:
add some comments.

Reply:
@gregmarr Can you speak to what is this `FunctionType` used for?

Reply:
It indicates which of the three types of function calls this node represents, a call to an external graph, a call to a function with a vararg argument, or a standard function.